controller/contactusController.js

```diff
@@ -1,7 +1,28 @@
 // dbClient = require('../dbConnection.js');
 
-const contactus = {
-
-};
-
-module.exports = contactus;
\ No newline at end of file
+const contactus = async (req, res) => {
+    try {
+      const { name, email, message } = req.body;
+      if (!name) {
```

Comment:
This isn't a problem but maybe we can discuss using a library for these validations something like [this](https://joi.dev/api/?v=17.12.2) to define a schema for the request.

Reply:
resolved

Reply:
I guess we can have a chat at next meeting to work out how we want to do validations moving forward